rdrf/report/report_configuration.py

```diff
@@ -120,6 +118,19 @@ def get_patient_fields():
                     'email': 'Email',
                     'selfPatientId': 'Self Patient ID'
                 }
-            }
+            },
+
         }
     }
+
+    if EmailNotification.objects.is_registry_subscribable(all_registries):
+        demographic_model['demographic_model'].update({
+            'emailNotifications': {
+                'label': 'Email Notification',
```

Comment:
Rename to `'Email Notifications'` to keep consistent with naming of other reportable model labels